### DISCUSSION TITLE
cors issue

### DISCUSSION THREAD

**Discussion by darbok** [supabase/supabase#20917] [Questions]
If i have in my headers 
"Access-Control-Allow-Credentials": "true",
    "Access-Control-Allow-Origin": '*',
    "Access-Control-Allow-Headers": 'authorization, x-client-info, apikey, content-type',

then why do I get
No 'Access-Control-Allow-Origin' header is present on the requested resource.

**Comment by GaryAustin1** [comment]
You might want to provide some context on what this pertains to?  What are you doing with these headers?

**Comment by darbok** [comment]
I was trying to post to my database table, I got that error in the console. 

Sent from Yahoo Mail on Android 
 
  On Wed, Jan 31, 2024 at 11:24 PM, ***@***.***> wrote:   


You might want to provide some context on what this pertains to? What are you doing with these headers?
